cloud/scope/powervs_cluster_test.go

```diff
@@ -1093,6 +1095,397 @@ func TestGetResourceGroupID(t *testing.T) {
 	}
 }
 
+func TestReconcileLoadBalancers(t *testing.T) {
+	var (
+		mockVpc  *mock.MockVpc
+		mockCtrl *gomock.Controller
+	)
+
+	setup := func(t *testing.T) {
+		t.Helper()
+		mockCtrl = gomock.NewController(t)
+		mockVpc = mock.NewMockVpc(mockCtrl)
+	}
+
+	teardown := func() {
+		mockCtrl.Finish()
+	}
+
+	t.Run("When LoadBalancer ID is set and GetLoadbalancer returns error", func(t *testing.T) {
+		g := NewWithT(t)
+		setup(t)
+		t.Cleanup(teardown)
+
+		clusterScope := PowerVSClusterScope{
+			IBMVPCClient: mockVpc,
+			IBMPowerVSCluster: &infrav1beta2.IBMPowerVSCluster{
+				Spec: infrav1beta2.IBMPowerVSClusterSpec{
+					LoadBalancers: []infrav1beta2.VPCLoadBalancerSpec{
+						{
+							ID: core.StringPtr("test-lb-instanceid"),
```

Comment:
Previously we were using core package for pointer conversion, Lately we are using `"k8s.io/utils/ptr"` so lets use ptr.To()
(Wherever applicable in the file)